src/hotspot/os/linux/osContainer_linux.cpp

```diff
@@ -137,9 +137,9 @@ jlong OSContainer::pids_current() {
   return cgroup_subsystem->pids_current();
 }
 
-void OSContainer::print_container_helper(outputStream* st, jlong j, const char* metrics) {
+void OSContainer::print_container_helper(outputStream* st, jlong j, const char* metrics, boolean limit) {
   st->print("%s: ", metrics);
-  if (j > 0) {
```

Comment:
I agree with @iklam. The fix for this seems to be to change the condition here to `j >= 0`.